doc/modules/ROOT/pages/index.adoc

```diff
@@ -25,37 +25,104 @@ In this guide you will learn how to deploy Hazelcast Kubernetes cluster and conn
 All the code used in this guide is available in https://github.com/hazelcast-guides/kubernetes-external-client.git[here].
 To download it, execute the following command.
 
-[source, shell]
+[source, shell script]
```

Comment:
Using `[source, shell]` would do the same. `script` keyword has no effect here as it's not needed to be removed. Just reminding for further updates. You can see the whole highlighting list here: https://github.com/hazelcast-guides/guides-ui/blob/master/src/js/vendor/highlight.bundle.js

Reply:
I see, thank you! My IDE, however, understands only `shell script`, will have to configure it